nextcord/application_command.py

```diff
@@ -540,14 +691,26 @@ def payload(self) -> dict:
             ret["options"] = [argument.payload for argument in self.options.values()]
         return ret
 
+    @property
+    def cog_application_before_invoke(self) -> Optional[ApplicationHook]:
+        """Returns the cog_application_before_invoke method for the cog that this command is in. Returns ``None`` if not the method is not found."""
+        if not self._self_argument: return None
+
+        return ClientCog._get_overridden_method(self._self_argument.cog_application_before_invoke)
+
+    @property
+    def cog_application_after_invoke(self) -> Optional[ApplicationHook]:
+        """Returns the cog_application_after_invoke method for the cog that this command is in. Returns ``None`` if not the method is not found."""
+        if not self._self_argument: return None
+
+        return ClientCog._get_overridden_method(self._self_argument.cog_application_after_invoke)
+
     # Methods that can end up running the callback.
 
     async def call_autocomplete(self, state, interaction: Interaction, option_data: List[Dict[str, Any]]) -> None:
         """|coro|
-
         This will route autocomplete data as needed, either handing it off to subcommands or calling one of the
         autocomplete callbacks registered.
-
         Parameters
```

Comment:
Again these seem to have been removed without reason making the code less readable?